src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationOptions.cs

```diff
@@ -101,6 +102,11 @@ internal IEnumerable<IKeyValueAdapter> Adapters
         /// </summary>
         internal bool IsKeyVaultRefreshConfigured { get; private set; } = false;
 
+        /// <summary>
+        /// A timeout to wait before initiating a request to the next App Configuration replica in parallel.
+        /// </summary>
+        internal TimeSpan ParallelRetryInterval { get; set; } = TimeSpan.FromSeconds(30);
```

Comment:
>TimeSpan.FromSeconds(30); [](http://example.com/codeflow?start=64&length=25)

This can be a constant so that its easy to find the default/min value in future.